### ISSUE TITLE
Target expression error

### ISSUE THREAD

**Issue by sqlmiles** [jsdelivr/n8n-nodes-globalping#4]
I'm trying to use an expression in DNS Measurement Target and i get an error: `Invalid target format: https://={{ $json['domain'] }}`

**Comment by jimaek** [comment]
Hey we're looking into this

**Comment by jimaek** [comment]
Please check now with the new version 1.0.4

**Comment by sqlmiles** [comment]
Hi, after updating the plugin I got the same type of error:

```
{
  "errorMessage": "Invalid target format: https://={{ $json.domain }}",
  "errorDetails": {},
  "n8nDetails": {
    "n8nVersion": "1.68.0 (Self Hosted)",
    "binaryDataMode": "default",
    "stackTrace": [
      "Error: Invalid target format: https://={{ $json.domain }}",
      "    at parseMeasurementTarget (/home/node/.n8n/nodes/node_modules/@globalping/n8n-nodes-globalping/nodes/Globalping/GenericFunctions.ts:68:3)",
      "    at Object.execute (/home/node/.n8n/nodes/node_modules/@globalping/n8n-nodes-globalping/nodes/Globalping/v1/GlobalpingV1.node.ts:145:45)",
      "    at Workflow.runNode (/usr/local/lib/node_modules/n8n/node_modules/n8n-workflow/dist/Workflow.js:722:42)",
      "    at /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/WorkflowExecute.js:704:66",
      "    at /usr/local/lib/node_modules/n8n/node_modules/n8n-core/dist/WorkflowExecute.js:1134:20"
    ]
  }
}
```

testing Workflow: [GlobalpingTest.json](https://github.com/user-attachments/files/17893022/GlobalpingTest.json)

**Comment by eyepokes** [comment]
Hi @sqlmiles, I'll recommend you to completely remove the node at `settings/community-nodes`, then install it again, it will be 1.0.4
<img width="727" alt="image" src="https://github.com/user-attachments/assets/23c75203-2991-45b8-aad3-07c9337887e6">
 then restart the n8n instance.
 The issue somehow related to n8n unable to update community nodes issue. https://github.com/n8n-io/n8n/issues/11840
 it will be solved in the 1.69 n8n version.
 
<img width="1172" alt="image" src="https://github.com/user-attachments/assets/58ae7603-f5d2-4122-ad13-020a85060e42">

**Comment by sqlmiles** [comment]
Before posting my previous comment I did an uninstall-install update but did not restart the n8n instance. After restarting everything works fine. 

Thanks for that and for the great software you created.
